packages/moonwalk/PKGBUILD

```
@@ -0,0 +1,36 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=moonwalk
+pkgver=v1.0.0.r18.g68d5be1
+pkgrel=1
+pkgdesc='Cover your tracks during Linux Exploitation by leaving zero traces on system logs and filesystem timestamps.'
+arch=('x86_64' 'aarch64')
+groups=('blackarch' 'blackarch-exploitation' 'blackarch-recon')
```

```suggestion
groups=('blackarch' 'blackarch-exploitation')
```

Doesn't seem related to recon at all.

---

> Doesn't seem related to recon at all.

Hello @noraj In the [README.md](https://github.com/mufeedvh/moonwalk) of the project, "Reconnaissance" is one of the features of the tool:
```
Reconnaissance: To save the state of system logs, moonwalk finds a world-writable path and saves the session under a dot directory which is removed upon ending the session.
```
Do you think we must include that BA group or not?

---

https://github.com/BlackArch/blackarch/pull/3766#discussion_r1160972220

---

`blackarch-recon` removed.